reconcile/utils/terrascript_aws_client.py

```diff
@@ -1181,6 +1185,64 @@ def populate_vpc_peerings(self, desired_state):
                     tf_resource = aws_route(route_identifier, **values)
                     self.add_resource(infra_account_name, tf_resource)
 
+    def populate_vpc_requests(self, vpc_requests: Iterable[VPCRequest]) -> None:
+        for request in vpc_requests:
+            # The default values here come from infra repo's module configuration
+            values = {
+                "source": "terraform-aws-modules/vpc/aws",
+                "version": "5.7.1",
```

Comment:
Okay. What was the bug? Can we make this a variable that get's passed in from integration with a comment linked to the bug explaining the hard coded value?

Reply:
+1 to Olivia's suggestion. Also this is the module's version, not the same as the provider version

```yaml
$schema: /aws/account-1.yml

name: some-account
# ...
providerVersion: 5.39.1
providerModulesVersions:
  terraform-aws-modules/vpc/aws: 5.7.1
```

I think it improves maintainability, for instance this would enable us to test a new version in a stage account first

Reply:
So I'm okay with extract this as a variable first then create a ticket to track necessary changes for  `providerModulesVersions:`. What do we think?